runtime/compilers/rillv1/parse_connector.go

```diff
@@ -17,20 +17,19 @@ func (p *Parser) parseConnector(node *Node) error {
 		return err
 	}
 
+	name := tmp.Name
+	if name == "" {
+		name = node.Name
+	}
```

Comment:
Can be removed because it's already done in `parseStem`: https://github.com/rilldata/rill/blob/b46a80ad941905f99176c3d3c22d445a51472b16/runtime/compilers/rillv1/parse_node.go#L252